# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/avr32/package/*/hotfix-std_cstdio.h.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-4.2.2/libstdc++-v3/include/c_std/std_cstdio.h.vanilla	2021-10-06 22:25:54.802431854 +0200
+++ gcc-4.2.2/libstdc++-v3/include/c_std/std_cstdio.h	2021-10-06 22:26:12.686431141 +0200
@@ -136,7 +136,7 @@
   using ::sprintf;
   using ::sscanf;
   using ::tmpfile;
-  using ::tmpnam;
+  //using ::tmpnam;
   using ::ungetc;
   using ::vfprintf;
   using ::vprintf;
